playbooks/ursula-agent.yml

```diff
@@ -0,0 +1,11 @@
+##
+- hosts: all
+  gather_facts: no
+  user: root
+  tasks:
+  - name: download ursula-agent repo
+    git: repo=git@github.blueboxgrid.com:bluebox-tech/ursula-agent.git dest=/tmp/ursula-agent accept_hostkey=yes
```

Comment:
How do we expect git authentication to work, this uses SSH host keys. Are we adding each node's SSH keys to a GHE account? Also, does this belong in a public repo, since it adds a direct dependency on our private GHE environment?